src/main/java/com/airlines/model/CrewMember.java

```diff
@@ -18,6 +19,7 @@ private CrewMember(Builder builder) {
         this.position = builder.position;
         this.birthday = builder.birthday;
         this.citizenship = builder.citizenship;
+        this.crewId = builder.crewId;
     }
 
     public int getId() {
```

Comment:
In java code you make id and crewId as int type - 32 bit integer.
But in SQL as BIGINT (64 bit value , long).

You should change or SQL script to INT or Java code to long to match database data model and java model

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/6c712e9343ba7917559526bfde8cd3da3375f810